src/_locales/ja/messages.json

```diff
@@ -0,0 +1,496 @@
+{
+    // manifest.json
```

Comment:
It would be nice to adjust the indentation to two spaces. This is [a known issue](https://github.com/Lusito/web-ext-translator/issues/30). Otherwise I'll soon do the PR by myself to get it merged.

Reply:
I've fixed the indentation of messages.json.
Please check.